spanner/unit_tests/test_client.py

```diff
@@ -106,6 +106,43 @@ def test_constructor_credentials_wo_create_scoped(self):
         expected_scopes = None
         self._constructor_test_helper(expected_scopes, creds)
 
+    def test_admin_api_lib_name(self):
```

Comment:
Not worth just checking lib name on one of the existing (e.g. constructor) unit test cases?